packages/dashboard/src/customization/propertiesSections/lineAndScatterStyleSettings/section.tsx

```diff
@@ -93,82 +85,6 @@ const RenderLineAndScatterStyleSettingsSection = ({
     })
   );
 
-  const [query] = useProperty(
```

Comment:
this was all duplicated in the `aggregationSettings` folder 
so i have removed it and updated the other files to account for line chart instead